app/models/api_model.php

```diff
@@ -45,8 +48,9 @@
         $view = 'api';
```

Comment:
We'd have to check but I think we are not using the $view variable any more and it could be deleted.